trainee_template/react_series.amlg

```diff
@@ -329,15 +333,50 @@
 						derived_context_features
 					)
 				))
-			output_influential_cases (get details "influential_cases")
-			influential_cases (list)
-			all_influential_cases (list)
-			output_cap (get details "categorical_action_probabilities")
-			local_class_probabilities_map (assoc)
-			all_categorical_action_probabilities (list)
-			aggregated_categorical_action_probabilities (assoc)
 		))
 
+		;if this method is being called from batch_react_series, warnings will already have been declared in BatchReact
+		;otherwise declare warnings here if this is being called from the single react_series call
+		(if (= (null) warnings)
+			(declare (assoc warnings (assoc) ))
+		)
+
+		(declare (assoc
+			requested_details (filter (lambda (get details (current_value))) (indices details))
```

Comment:
same issue as the other request_details filter

Reply:
yep